Flick/Controllers/SettingsViewController.swift

```diff
@@ -0,0 +1,154 @@
+//
+//  SettingsViewController.swift
+//  Flick
+//
+//  Created by Lucy Xu on 8/20/20.
+//  Copyright © 2020 flick. All rights reserved.
+//
+
+import UIKit
+import FBSDKLoginKit
+
+class SettingsViewController: UIViewController {
+
+    // MARK: - Private View Vars
+    private let aboutButton = UIButton()
+    private let editProfileButton = UIButton()
+    private let headerView = UIView()
+    private let logoutButton = UIButton()
+    private let sendFeedbackButton = UIButton()
+    private let settingsTitleLabel = UILabel()
```

Comment:
i think making settings uibuttons is ok for now since we don't have that many, but in the future when we add more to settings we prob want to use a tableview

Reply:
Mhm, I just didn't know if we will be adding more to the settings page so I didnt want to make the page more complex that it has to be haha